src/assets/styles/toolbar.less

```diff
@@ -1,7 +1,8 @@
 ﻿#toolbar {
   height: 50px;
   right: 65px;
```

Comment:
Similarly there's a hard-coded height for `.sidetab-top` in sidebar.less that should be made a variable and reused here.